Whisper.net/Whisper.net.csproj

```diff
@@ -21,6 +21,10 @@
 		<None Include="..\LICENSE" Pack="true" PackagePath=""/>
 		<None Include="..\readme.md" Pack="true" PackagePath=""/>
 	</ItemGroup>
+
+  <ItemGroup>
+	  <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
+  </ItemGroup>
```

Comment:
Can we implement it without this to not add an unnecessary dependency for all users?

We know that callbacks will be called synchronous (no parallel callbacks will be called) so we can just use a list of TaskCompletionSources for the results.

Reply:
This provides `IAsyncEnumerable`, it is otherwise not available.

Reply:
Can we add it only for netframework in this case?
NetCore should have it.

Reply:
It is only available in .net standard 2.1 and newer. .net core does not have it until 3.1 iirc.

Reply:
got it, let's keep it like this for now in this case and for some release version I will think of breaking it so for someone with net3.1+ we won't have it as a dependency.